### DISCUSSION TITLE
fpsync progress status?

### DISCUSSION THREAD

**Discussion by lastb0isct** [martymac/fpart#57] [Ideas]
I saw a couple issues that were raised regarding progress of transfers.  I'm curious what the status of this is?

As of now is the only way to get detail of your transfer progress to add verbose options??

**Comment by martymac** [comment]
Hello,

There are two ways of getting transfer status with fpsync :

- using the verbose option, as you stated
- sending a SIGINFO signal to the main process (pressing CTRL^T on FreeBSD)

I've recently improved the overall status display by adding size information. This is already in master branch and will be present in next release.

Best regards,

Ganael.

**Comment by lastb0isct** [comment]
Thanks Ganael!  Using -vvv it doesn't really give me details but it does show how many jobs it has done.  Do I need to expand that further with more `v`?

**Comment by martymac** [comment]
Hello,

Using -vv is enough, you will not get more details by adding more -v here. You have to send a SIGINFO to the fpsync parent process if you want to get more details : latest code in master branch now makes SIGINFO show bytes/files/parts done.

Best regards,

Ganael.

**Comment by fleetwoodmac** [comment]
just as another option here, I use `multitail` to see what files are being copied and how fast in a new terminal tab conjunction to the ctrl+t SIGINO suggested. it's pretty messy but it gives me a rough idea of progress.

Example: start an Fpsync transfer and in another terminal tab run `multitail -Q 1 "/tmp/fpsync/log/job-id/*.stdout"`, where job-id is the run id that Fpsync give you for the ongoing transfer. -Q 1 is the query interval in seconds, adjust as you like. I am sure this can be tuned to make multi tail output more useful/clean, but it gets the job done.
